.travis.yml

```diff
@@ -41,10 +41,21 @@ jobs:
       env: aarch64
       arch: arm64
       script:
-        - sed -i 's/java_arch=32-bit$/java_arch=64-bit/' build-image/openhabian.conf
+        - sed -i 's/java_opt=Zulu32$/java_opt=Zulu64/' build-image/openhabian.conf
         - docker build -t openhabian/dev-openhabian -f Dockerfile.openhabian-aarch64-native .
         - docker run -i openhabian/dev-openhabian bash -c '/usr/local/bin/bats -r -f "dev-." .'
         - docker run -i openhabian/dev-openhabian bash -c '/usr/local/bin/bats -r -f "unit-." .'
+    - env: aarch64
+      arch: arm64
+      script:
+        # 64bit Azul Java on 64bit ARMv8 (RPi3,4) on 64bit OS on Travis Arm64 HW
+        - export HW=rpi4
+        - sed -i 's/java_opt=Zulu64$/java_opt=AdoptOpenJDK/' build-image/openhabian.conf
+        - sed -i 's/java_opt=Zulu32$/java_opt=AdoptOpenJDK/' build-image/openhabian.conf
+        - docker build -t openhabian/${HW}-openhabian -f Dockerfile.openhabian-aarch64-64bit .
```

Comment:
why did you base this on the 64bit OS use case ? Need this for arm32hf and eventually aarch64 on 32bit OS
(eventually change one of the existing cases else we'll end up with too many build runs)

Reply:
Yeah, I was just using a quick test case. Just grabbed a random example, I can change that out.